library/std/src/io/util.rs

```diff
@@ -19,7 +19,7 @@ pub struct Empty;
 
 /// Constructs a new handle to an empty reader.
 ///
-/// All reads from the returned reader will return `Ok(0)`.
+/// All reads from the returned reader will return [`Ok(0)`].
```

Comment:
are you sure this actually works?

Reply:
It seems to work with rust-analyzer's doc parser, but I can add those link aliases at the bottom of the doc if needed.
![image](https://user-images.githubusercontent.com/41439633/176548719-2e9e3707-f078-4e94-9ac1-41141314fd82.png)

Reply:
Hmm, ok - I'd prefer to test that rustdoc works, but I think I vaguely remember a PR allowing this so it's probably fine

Reply:
the rustdoc build failed, and i updated it to use aliasing this morning. :)